core/field_slider.js

```diff
@@ -180,10 +180,10 @@ Blockly.FieldSlider.prototype.addSlider_ = function(contentDiv) {
     this.readout_ = elements[1];
   }
   this.slider_ = new goog.ui.Slider();
-  this.slider_.setMoveToPointEnabled(false);
   this.slider_.setMinimum(this.min_);
   this.slider_.setMaximum(this.max_);
-  if (this.step_) this.slider_.setUnitIncrement(this.step_);
+  this.slider_.setMoveToPointEnabled(!this.step_);
+  if (this.step_) this.slider_.setBlockIncrement(this.step_);
```

Comment:
re: https://google.github.io/closure-library/api/goog.ui.Slider.html; unitIncrement is used but doesn't have a visible change in this case, it's for arrow keys / scroll wheel / etc. The value that is meaningful for us is 'blockIncrement' which is what's used for large jumps, including the 'move towards where you clicked by n amount'